### DISCUSSION TITLE
Alguém recebendo em Bitcoin?

### DISCUSSION THREAD

**Discussion by fforbeck** [DyegoCosta/trabalhando-remoto#179] [General]
Vi alguns casos de empresas de fora que pagam em Bitcoin para galera que trabalho remoto.
Alguém aqui já recebeu/recebe em Bitcoins e gostaria de compartilhar?

**Comment by ribeiroevandro** [comment]
Já passei por essa experiência trabalhando remoto para uma agência da Califórnia... Qual seria a sua dúvida sobre?

**Comment by kaiomagalhaes** [comment]
Eu recebo em btc hoje, de boas. A taxa é quase sempre maior que a do dolar,então compensa

**Comment by fforbeck** [comment]
@ribeiroevandro @kaiomagalhaes 
qual exchange vcs utilizam?

**Comment by kaiomagalhaes** [comment]
@fforbeck Foxbit é o que tem a melhor taxa, não é tão estavel quanto o mercado bitcoin, mas é tranquilo

**Comment by ribeiroevandro** [comment]
Também uso a Foxbit, fora esse problema de instabilidade que aconteceu recente, o envio do dinheiro para minha conta no Itaú que não é banco parceiro deles é MUITO rápido, todo problema que tive, eles resolveram prontamente, recomendo eles sempre...

**Comment by rubemz** [comment]
Eu uso https://localbitcoins.com. Funcionou muito bem pra mim e tu trata direto com o comprador. O sistema é baseado em reputação, então talvez demore um pouco para as tuas ofertas de venda de btc apareçam em primeiro no rank.

**Comment by ribeiroevandro** [comment]
@rubemz interessante esse site, não conhecia...

**Comment by its-felipe-almeida** [comment]
Vocês que recebem em bitcoin, como funciona a declaração do IR? Sai mais em conta do que receber em dolar, ou não?

**Comment by MiattoRocha** [comment]
Aproveitando essa Issue, gostaria de sugerir (pois também queria saber), sobre a possibilidade de usar ferramentas como  a Bitwage. https://www.bitwage.com/
Eu não fui a fundo, mas pelo que entendi, esse site lhe fornece uma conta em um banco norte americano e você passa esses dados pro empregador que te paga normalmente, no entanto, na sua conta Bitwage você terá bitcoins.
Também queria saber sobre declaração do IR e impostos com mais detalhes.

**Comment by ribeiroevandro** [comment]
@MiattoRocha sobre impostos e IR não sei te dizer com detalhes, mas eu tenho conta na Bitwage e contato direto com um representante deles via WhatsApp. Se quiser formular perguntas, posso encaminhar para ele ou caso ele autoriza, passo o contato para você, onde poderá tirar todas as suas dúvidas, assim como eu fiz antes de abrir minha conta, só esqueci desses detalhes que citou, rs

**Comment by MiattoRocha** [comment]
@ribeiroevandro bacana, no meu caso, eu ainda não trabalho pra nenhuma companhia fora do Brasil então, ainda não utilizaria esses serviços. Levantei essa bola pois, acho que muitos brasileiros ainda recebem via paypal ou outras formas que fazem eles perder bastante dinheiro comparado a essa transação bitcoin. Talvez, se for um serviço bacana, incluir essa possibilidade no texto do repositório, na área "Como receber do exterior".

**Comment by fforbeck** [comment]
@felipeaugusto1 @MiattoRocha 
Posso estar falando besteira, mas eu acredito que pra quem é PJ o IR é a mesma coisa, pois você não emite nota falando que recebeu em BTC. É o mesmo processo pra quem recebe em USD por exemplo. 
Você converte para R$, cai na conta e emite a nota de serviço. 

Acredito que a vantagem em receber em BTC ou qualquer outra cripto moeda é que o spread é bem menor. Além disso você tem a possibilidade de mandar o BTC para um cartão de débito, como [Advanced Cash](http://wallet.advcash.com/referral/360e6f81-c680-40c6-849b-67479d6ec72e) por exemplo, e utilizar o cartão com BTCs. 

Penso que nesse caso vc não irá declarar e nem emitir nota de serviço, pois está tudo em BTC mesmo. Em outras palavras, eu acredito que você precisará declarar e emitir nota somente do que você converteu para R$ e caiu na sua conta no banco.

**Comment by its-felipe-almeida** [comment]
muito boa explicação, @fforbeck . eu acredito que é mais ou menos por aí, também. mas eu perguntei porque ouço falar que existem várias regras pra declaração de bitcoin. por exemplo, se ganhar mais de 35k (em real) por mês, deve declarar e pagará imposto. se tiver 1k de bitcoin (em real), só precisa declarar, e etc. enfim, eu já vi várias coisas relacionadas a isso, mas assumo que não fui a fundo pra saber como que é.

**Comment by fernandomm** [comment]
Alguém conhece algum serviço que facilite a compra do bitcoin por parte da empresa no exterior?

Ex.: a empresa que trabalho quer me mandar 1.000 USD por bitcoin. Eles não são pessoas com conhecimento técnico.

Eu tenho a carteira ng4SJ3mRcGzFGY2DdmiZ48WNFX1mOC1k no Mercado Bitcoin. Eu coloco o número da minha carteira, a empresa que eu trabalho efetua o pagamento por cartão de crédito por uma interface web simples e os Bitcoins são creditados na minha carteira. Com as devidas cobranças de taxa do cartão e transferência do BTC é claro.

**Comment by pablo-sg-pacheco** [comment]
O bitwage faz isso.
Eu inclusive estou usando.

Vou colar um trecho de uma conversa com um amigo meu pra te explicar mais
ou menos

Eh pq eu uso uma ferramenta chamada Bitwage. Eh mais ou menos assim:

O meu empregador deposita o dinheiro (euro no caso) na conta da bitwage
mais proxima dele, sem pagar taxas.
Logo que o bitwage recebe o dinheiro, ele converte pra bitcoin e passa pra
mim como eu quiser. Uma parte eu escolho receber em real e a outra parte eu
recebo direto em bitcoin mesmo. O que eu quero receber em real, eles ja
convertem pra mim. E a parte que eu quero receber em bitcoin eles mandam
direto pra minha foxbit.

Eles cobram 1%

Em 14 de dez de 2017 18:08, "Fernando Morgenstern" <notifications@github.com>
escreveu:

Alguém conhece algum serviço que facilite a compra do bitcoin por parte da
empresa no exterior?

Ex.: a empresa que trabalho quer me mandar 1.000 USD por bitcoin. Eles não
são pessoas com conhecimento técnico.

Eu tenho a carteira ng4SJ3mRcGzFGY2DdmiZ48WNFX1mOC1k no Mercado Bitcoin. Eu
coloco o número da minha carteira, a empresa que eu trabalho efetua o
pagamento por cartão de crédito por uma interface web simples e os Bitcoins
são creditados na minha carteira. Com as devidas cobranças de taxa do
cartão e transferência do BTC é claro.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/DyegoCosta/trabalhando-remoto/issues/43#issuecomment-351821950>,
or mute the thread

**Comment by fernandomm** [comment]
Obrigado pela resposta @pablo-sg-pacheco, ajudou muito. Você sabe se é possível receber o dinheiro em uma conta de pessoa jurídica? Ou para isso eu precisaria transferir para BTC e depois sacar no Foxbit?

**Comment by pablo-sg-pacheco** [comment]
Tranquilo :)
Nao tenho certeza mas acho que da sim. Basta cadastrar a conta empresa

**Comment by fernandomm** [comment]
Acabei mandando um email para eles e confirmaram que é possível.

```
Hi Fernando,

Yes we could do this. We might ask for extra verification to make sure the account belongs to you though.
```

Acho que bitwage com o membership premium pode ser uma boa opção a partir de certo valor já que parece que não tem a taxa de 1%.

Vou tentar receber um valor menor para confirmar se realmente chega.

Também vou verificar com o meu contador como fica a questão do ISS, já que hoje não preciso pagar.

**Comment by pablo-sg-pacheco** [comment]
Blz. Boa.
Esse ano eu recebi só como pessoa física. Acabei de conseguir abrir empresa
pela Contabilizei. Depois eu conto a minha experiencia de receber como
empresa



2017-12-15 8:53 GMT-02:00 Fernando Morgenstern <notifications@github.com>:

**Comment by fernandomm** [comment]
Segue minha experiência com o Bitwage. Fiz o cadastro e enviei os documentos. Demorou 3 dias úteis para aprovarem.

Já assinei o serviço premium ( 15,99 USD/mês ) que retira a taxa de $10 para saque e a taxa de 1%, além de reduzir os prazos de pagamento.

Como teste, foi feito o envio de 1000 euros. Escolhi receber 25% em bitcoin e 75% direto na conta corrente.

A transferência por parte da empresa que forneço serviços foi feita na terça a noite e apareceu na quinta no sistema da Bitwage. Na própria quinta-feira, os bitcoins foram transferidos. A transferência para a conta corrente foi feita na sexta-feira porém na quinta-feira o valor em reais já tinha sido fixado.

Os **750 euros** que entraram direto na conta foram convertidos para **R$ 2912,73**. Ou seja, a cotação foi de 3,88. O euro fechou nesse dia a 3,92. Então da uma diferença de 1%, bem menor do que os bancos costumam utilizar.

Os **250 euros** transferidos por bitcoin foram convertidos em **0.0182363 BTC**. Caso alguém tenha interesse para ver a cotação, a transferência foi feita em torno das 18:00 de quinta-feira.

De forma geral eu gostei do serviço deles e a cotação/taxas são muito interessantes. O único problema que vejo é que eles não são um banco. Então nesses 2-3 dias que o dinheiro fica com eles, se acontecer um problema, você pode simplesmente perder ele e não tem muito o que fazer. Seria como o Paypal bloquear sua conta do nada, o que acontece com frequência. O Bitwage informa no site que tem seguro sobre esses valores mas não fica claro como esse seguro funciona.

Eu pretendo seguir utilizando o serviço deles. Mas ao invés de transferir valores grandes ( 5000+ USD ), já combinei com a minha empresa para que ela envie aos poucos. Ex.: de 1000 em 1000. Assim sempre vou aguardar receber um pagamento para pedir o outro. Se acontecer alguma coisa, pelo menos não vou perder grandes valores.


* **Recebimento do Bitcoin**

Essa parte não tem relação com o Bitwage. Mas decidi colocar ela para informar os valores completos. Eu não tenho conta na Foxbit como PJ e eles estão fechados para cadastro devido a alta demanda. Então tive que receber pelo Mercado Bitcoin, onde já tinha uma conta PJ.

Como informei, chegaram 0.0182363 BTC ( 250 euros ) e tive que esperar cerca uma hora para que as 6 confirmações da rede Bitcoin ocorressem e o Mercado Bitcoin aceitasse o depósito. Converti eles para reais. Deu **R$ 1033,76** bruto e foi cobrado **R$ 7,23** de comissão pelo Mercado Bitcoin. Total líquido: **R$ 1026,52**.

Então fiz o pedido de saque para conta corrente, sendo que o Mercado Bitcoin cobra R$ 2,90 + 1,99%. Em valores: a comissão foi de **R$ 23,32** e recebi na conta **R$ 1003,20**.

Ou seja, em relação aos 250 euros, da uma cotação de 4,0128.

Através da Foxbit seria ainda melhor, visto que as taxas deles são menores. Porém é importante lembrar das variações no valor do Bitcoin que acontecem em pouco tempo. Para evitar grandes surpresas, é interessante ficar em cima assim que a transferência ocorre para vender e sacar o mais rápido possível.

**Comment by MiattoRocha** [comment]
Valeu @fernandomm por compartilhar essa experiência, foi bem esclarecedor os processo! Acredito que vai ajudar bastante quem tem dúvidas nisso.

**Comment by joaoricardo000** [comment]
@fernandomm cara, tais recebendo o dinheiro direto como pessoa física, ou como PJ?  
Minha dúvida é se com o Bitwage posso receber na minha conta PJ para fazer toda a declaração certinha.

**Comment by fernandomm** [comment]
@joaoricardo000 PJ. O depósito foi feito diretamente na conta da minha empresa.

**Comment by brunolm** [comment]
Como que faz para ligar a conta do Foxbit no Bitwage?

E como devo declarar a renda?

**Comment by pablo-sg-pacheco** [comment]
Bom, pra declarar eu ainda n sei direito.
Mas vc n liga a conta da foxbit diretamente. Vc simplesmente cadastra um endereço bitcoin

**Comment by caarlos0** [comment]
Meu futuro empregador quer pagar em BTC, nesse caso pelo que entendi o bitwage nao faz sentido, é isso?

Alguém sabe quais exchanges brazukas fazem depósito pra conta pessoa juridica?

**Comment by fforbeck** [comment]
@caarlos0 
Realmente não precisa utilizar a bitwage nesse caso. 

Quanto a exchange, a Foxbit faz: https://foxbit.zendesk.com/hc/pt-br/articles/115003918143-Conta-Pessoa-Jur%C3%ADdica

Só não sei como está o tempo de abertura da conta.

**Comment by fabioluiz1** [comment]
A bitwage pode ser vantajosa, pois vc tem a opção de escolher a distribuição do pagamento, por exemplo, 60% em transferência bancária e 40% em bitcoin.

 Portanto, vc não precisaria transferir os bitcoins para uma exchange e negociá-los para poder realizar o saque.

A taxa de serviço do bitwage tb é ligeiramente menor (1%) que a taxa de saque da Foxbit (1,39%).

**Comment by caarlos0** [comment]
BitcoinTrade também aceita, mas tem que abrir a conta no CNPJ.

R$ 4,90 por saque + 0,9%

parece barato...

**Comment by afa7789** [comment]
Pessoal vcs ainda usam Btc para receber ? conhecem alguma outra alternativa além da BitWage ?
